packages/mobile/package.json

```
@@ -88,7 +88,7 @@
     "date-fns": "^2.12.0",
     "dot-prop-immutable": "^1.4.0",
     "es6-promisify": "^6.1.1",
-    "ethereumjs-util": "^6.2.0",
+    "ethereumjs-util": "^7.3.1",
```

The last published version is [`7.1.3`](https://www.npmjs.com/package/ethereumjs-util)
```suggestion
    "ethereumjs-util": "^7.1.3",
```